main.nf

```diff
@@ -1013,9 +1027,9 @@ process Register_T1 {
 
     output:
     set sid, "${sid}__t1_warped.nii.gz" into t1_for_seg
-    file "${sid}__output0GenericAffine.mat"
+    set sid, "${sid}__t1_warped.nii.gz", "${sid}__output0GenericAffine.mat",
+        "${sid}__output1InverseWarp.nii.gz" into t1_for_freesurfer_reg
     file "${sid}__output1InverseWarp.nii.gz"
-    file "${sid}__output1Warp.nii.gz"
```

Comment:
Hi, I'm wondering why `output1Warp` is removed from the file output? Thanks.

Reply:
Hi @liuzhenqi77 thanks for the comment. I will fix this.